navigation.yaml

```
@@ -328,6 +328,9 @@ download:
           path: /download/iot/up-squared-iot-grove-server
           hidden: True
 
+         - title: Installation media
```

Sorry, this was clearly my fault, and broke the test. Should be:

```suggestion
         - title: Installation media
```